<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>agregar</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="botonmostrar">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>MOSTRAR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_blue">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>133</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_voltaje">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>133</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_posx">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>133</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_posy">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>133</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_red">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>133</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_green">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>133</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="botonagregarfinal">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>280</y>
        <width>95</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>AGREGAR FINAL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonagregainicio">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>98</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>AGREGAR INICIO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>45</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 700 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>BLUE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 0);
font: 700 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>GREEN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>35</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 700 12pt &quot;Arial&quot;;
gridline-color: rgb(85, 0, 255);</string>
      </property>
      <property name="text">
       <string>RED</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>93</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>POSICION Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>93</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>POSICION X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>VOLTAJE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>16</width>
        <height>18</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>213</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>AGREGAR NEURONAS:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>134</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>NEURONAS</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>buscar</string>
     </attribute>
     <widget class="QPushButton" name="boton_buscar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boton_mostrartabla">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar tabla</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_buscar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 700 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Buscar Neurona</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>371</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGraphicsView" name="grafico">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>621</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="dibujar_pushButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>280</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>DIBUJAR NEURONA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="limpiar_pushButton_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LIMPIAR</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuarchivo">
    <property name="title">
     <string>archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionguardar"/>
    <addaction name="actionabrir"/>
   </widget>
   <addaction name="menuarchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionguardar">
   <property name="text">
    <string>guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionabrir">
   <property name="text">
    <string>abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
